package.json

```diff
@@ -11,10 +11,10 @@
   "files": [
     "index.js"
   ],
-  "dependencies": {},
```

Comment:
are these package changes necessary?

Reply:
Adding koa is necessary for the type information